csharp/ql/src/Security Features/CWE-016/ASPNetMaxRequestLength.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+The <code>maxRequestLength</code> attribute sets the limit for the input stream buffering threshold in KB. Attackers can use large files to cause denial-of-service attacks.
+</p>
+
+</overview>
+<recommendation>
+
+<p>
+The recommended value is 4096 but you should try setting it as small as possible according to business requirements.
```

Comment:
Perhaps change this sentence to:

The recommended value is 4096 _KB_, but you should set it as small as possible according to business requirements.

(Note, include units)